packages/serverless-cdk-plugin/package.json

```diff
@@ -40,7 +40,11 @@
   "dependencies": {
     "@serverless/typescript": "^3.25.0",
     "@types/serverless": "^3.12.8",
+    "aws-cdk-lib": "^2.54.0",
+    "constructs": "^10.1.187",
+    "json-schema-to-ts": "^2.6.2",
     "lodash": "^4.17.21",
+    "serverless": "^3.25.1",
```

Comment:
they are already in peerDependencies
eslint should be configured to accept that